ribbon_test_environment/README.rst

```diff
@@ -0,0 +1,25 @@
+Ribbon Test Environment
```

Comment:
For README, please use this template
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst